analysis/dataflow/function_summary_graph.go

```diff
@@ -1295,10 +1292,14 @@ func ppEdge(w io.Writer, n GraphNode, c EdgeInfo, arrow string) {
 	if c.Index >= 0 {
 		prefix += "#" + strconv.Itoa(c.Index)
 	}
-	if len(c.RelPath) > 0 {
-		prefix += "@"
-		for inPath, outPaths := range c.RelPath {
-			for outPath := range outPaths {
+	prefixed := false
+	for inPath, outPaths := range c.RelPath {
+		for outPath := range outPaths {
+			if inPath != "" || outPath != "" {
+				if !prefixed {
+					prefix += "@"
```

Comment:
Doesn't need to be addressed in this PR but I'm wondering if we should come up with a more efficient representation for paths? Strings are relatively expensive to update because they are immutable. Maybe it would be more efficient to use `[][]byte` or `[]string` instead?